### DISCUSSION TITLE
Can't launch

### DISCUSSION THREAD

**Discussion by GlandusFalngus** [Sixze/ALS-Refactored#458] [Q&A]
I read Quick Start, but not only am I unable to start this project, but when I start any other I write this

"The following modules are missing or built with a different engine version:

  ALS
  ALSCamera
  ALSExtras
  ALSEditor

Engine modules cannot be compiled at runtime. Please build through your IDE."

**Comment by MrWilsonWilson** [comment]
right click your .uproject, generate visual studio files, open the new .sln file, and compile + launch from there !
